circle.yml

```diff
@@ -0,0 +1,29 @@
+---
+machine:
+  environment:
+    PATH: "${PATH}:${HOME}/${CIRCLE_PROJECT_REPONAME}/node_modules/.bin"
+  node:
+    version: 8.2.0
+
+dependencies:
+  override:
+    - yarn global add codecov
+    - yarn install --no-progress --no-emoji --ignore-engines
+  cache_directories:
+    - "node_modules"
+    - ~/.yarn
+    - ~/.yarn-cache
```

Comment:
I think the correct path is: `- ~/.cache/yarn`

Reply:
Ok will change that ;)